controls/radrichtextbox/features/clipboard-support.md

```diff
@@ -75,19 +60,23 @@ When building UI for the RichTextBox, the respective commands can be used:{% if
 	</telerik:RadToolBar>
 {{endregion}}
 
-{% endif %}
-
 >Note that the attached __telerik:RadRichTextBoxRibbonUI.RichTextCommand__ property works only with Ribbon buttons. With regular RadButtons and other buttons, you should use their __Command__ property instead.
           
+## Key Bindings
+
+>tipIn order to copy, paste or cut, the standard keyboard shortcuts can also be used - __Ctrl + C__, __Ctrl + V__, __Ctrl + X__.
+          
+
+To learn more about the default key-bindings of the editor and ways to override them, you can refer to this article: [Keyboard Support]({%slug radrichtextbox-features-keyboard-support%}).

```

Comment:
Can we say "you can refer to the [Keyboard Support] article", instead of "to this article: [...]"?

Reply:
Yep, thanks